### PR TITLE
Add the readme file and link to the weekly updates

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@iamxy @c4pt0r @siddontang @shenli PTAL.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
++ [September 05 ~ September 11](./_posts/2016-09-12-tidb-weekly.md)
++ [August 29 ~ September 04](./_posts/2016-09-05-tidb-weekly.md)
++ [August 22 ~ August 28](./_posts/2016-08-29-tidb-weekly.md)
++ [August 13 ~ August 21](./_posts/2016-08-22-tidb-weekly.md)
++ [August 05 ~ August 12](./_posts/2016-08-12-tidb-weekly.md)
++ [July 30 ~ August 05](./_posts/2016-08-05-tidb-weekly.md)
++ [July 23 ~ July 29](./_posts/2016-07-29-tidb-weekly.md)
++ [July 17 ~ July 22](./_posts/2016-07-23-tidb-weekly.md)
+
+
```
Comment (siddontang) [inline README.md]: remove following blank lines.

**Comment by iamxy** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**